pkg/objects/storage/providers/database/provider.go

```diff
@@ -42,10 +49,17 @@ func (p *Provider) Upload(ctx context.Context, reader io.Reader, opts *storagety
 		return nil, err
 	}
 
+	mimeType := http.DetectContentType(data)
+
+	fileMutation := client.File.UpdateOneID(fileID).
+		SetFileContents(data)
```

Comment:
I don't think we should do this; persisting files (regardless of size) as b64 encoded incurs a lot of unnecessary penalties; if there are reasons we want to have the image as b64 encoded so that we can respond with the entire contents in the response payload, we should encode the image from the byte field and inject it into the API response payload